src/commonMain/kotlin/character/classes/druid/talents/Vengeance.kt

```diff
@@ -0,0 +1,19 @@
+package character.classes.druid.talents
+
+import character.Talent
+
+/**
+ *
+ */
+class Vengeance(currentRank: Int) : Talent(currentRank) {
```

Comment:
I would recommend implementing this in a similar way to warlock Ruin: https://github.com/marisa-ashkandi/tbcsim/blob/main/src/commonMain/kotlin/character/classes/warlock/talents/Ruin.kt